packages/fa-icon-chooser/src/components/fa-icon-chooser/fa-icon-chooser.tsx

```diff
@@ -596,7 +596,7 @@ export class FaIconChooser {
       .join(' ');
   }
 
-  shouldEmitSvgData(prefix: string): boolean {
+  shouldEmitSvgData(): boolean {
```

Comment:
Could you explain why you changed the logic of this function?

Previously, it would tell you, for a given prefix, if icons in that prefix were allowed to be embedded.

Now it seems to say: "if _any_ prefix is allowed to be embedded, then all of them are", because (apart from the override), it always returns `true` when `this.embedSvgPrefixes` is non-empty.

I don't think that's how this should work. But maybe you've thought of something I haven't?

Reply:
The reason I removed:

For pro lite and pro lite plus users `this.embedSvgPrefixes` will be empty.
So I thought "why bother" passing something in, we already have what we need.

But perhaps, it is better to be explicit here.  So changing back.

Reply:
ok, yeah, I get it.